jupyter-tensorflow-notebook/Dockerfile

```
@@ -1,7 +1,8 @@
 
 FROM jupyter/tensorflow-notebook:ubuntu-22.04
 
-RUN pip install jupyter-server-proxy tensorflow tensorboard --no-cache-dir
+# pip install required packages
+RUN pip install jupyter-server-proxy tensorflow==2.12.0 tensorboard --no-cache-dir 
 RUN pip install gym==0.25.2 pygame --no-cache-dir
```

Most of this should not be required.

See https://github.com/stfc/cloud-docker-images/blob/master/jupyter-cil-notebook/Dockerfile:

```suggestion
RUN mamba install -y -c conda-forge gym pygame \
  jupyter-server-proxy \
  && mamba clean -a -y -f \
  && fix-permissions "${CONDA_DIR}" \
  && fix-permissions "/home/${NB_USER}"
```